tools/js-tools/git-hooks/pre-push-hook.js

```diff
@@ -10,6 +10,17 @@ const chalk = require( 'chalk' );
  */
 function checkChangelogFiles() {
 	console.log( chalk.green( 'Checking if changelog files are needed. Just a sec...' ) );
+
+	// Bail if we're pushing to a release branch, like boost/branch-1.3.0
+	let currentBranch = spawnSync( 'git', [ 'branch', '--show-current' ] );
+	currentBranch = currentBranch.stdout.toString().trim();
+	const branchReg = /.*\/branch-(\d+).(\d+).(\d+)/; // match example: jetpack/branch-1.2.3
```

Comment:
Shouldn't that patch number be optional? I believe that we don't use that patch number for most Jetpack releases.

```suggestion
	const branchReg = /.*\/branch-(\d+).(\d+)(.(\d+))?/; // match example: jetpack/branch-1.2.3
```

Reply:
Nice catch, you're right. Committed suggestion.